src/applications/check-in/pre-check-in/pages/Error/index.jsx

```diff
@@ -51,16 +53,34 @@ const Error = () => {
     </>
   );
 
+  // use type param to set unique error messages
+  const getErrorMessagePropsByType = () => {
```

Comment:
If we're going to go with the params method, here is where we could extend Error to pass alternate unique messages to ErrorMessage